### DISCUSSION TITLE
No access to github copilot even though my application was approved (Months ago!)

### DISCUSSION THREAD

**Discussion by kevinp972** [community/community#143916] [GitHub Education]
### Hi everyone,

approved Submitted Nov 8, 2024
Congrats! Your request for [@kevinp972](https://github.com/kevinp972) was approved and your benefits will be available [soon](https://github.com/orgs/community/discussions/111352#user-content-how-long-after-ive-been-approved-will-i-receive-my-academic-benefits). But still, I have no access to Github Copilot

**Comment by huddmax** [comment]
Hi there! 👋 

Congrats on getting approved for the GitHub Student Pack! 🎉 To access GitHub Copilot, you might need to activate it directly on the [GitHub Student Developer Pack page](https://education.github.com/pack).

Here’s what to do:

1. Visit the [GitHub Student Developer Pack page](https://education.github.com/pack).
2. Scroll down to find the GitHub Copilot benefit, and follow the steps to activate it.
3. If Copilot is still unavailable after activating, try logging out and back in or checking for any additional instructions in your email.

If you’re still having trouble, GitHub Education Support can help sort it out. Enjoy using Copilot! 😊

**Comment by davevad93** [comment]
Hi @kevinp972 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @kevinp972 was approved and your benefits are now available.`

However it usually takes a few days, or a couple of weeks at most, in your case since you're still waiting for months, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.
